src/components/MonthlyGood/MonthlyGoodSelector.vue

```diff
@@ -30,7 +30,7 @@
 							'click-mgpromo-cause',
 							option.marketingName
 						]"
-						@click="selectCause(option)"
+						@click.stop="selectCause(option)"
```

Comment:
Stops propagation and prevents the click-outside handler for firing since these elements are absolutely positioned and are technically 'outside'